src/openms/source/ANALYSIS/OPENSWATH/TransitionTSVFile.cpp

```diff
@@ -1452,7 +1678,11 @@ namespace OpenMS
         + (String)it.detecting_transition     + "\t"
         + (String)it.identifying_transition   + "\t"
         + (String)it.quantifying_transition   + "\t"
-        + ListUtils::concatenate(it.peptidoforms, "|");
+        + ListUtils::concatenate(it.peptidoforms, "|") + "\t"
+        + ListUtils::concatenate(it.OligoName, "|") + "\t"
+        + (String)it.NuctideSequence + "\t"
+        + (String)it.FullNuctideName
+        ;
```

Comment:
@jcharkow can it cause issues if extra columns are added?

Reply:
might be relevant if it is carried down to the downstream tools (pyprophet etc.)